src/pages/builders.js

```diff
@@ -0,0 +1,251 @@
+import React from 'react';
+import cx from 'classnames';
+import SEO from '../components/Seo';
+import { Button } from '@newrelic/gatsby-theme-newrelic';
+import PageLayout from '../components/PageLayout';
+import ExternalLink from '../components/ExternalLink';
+import FeatherIcon from '../components/FeatherIcon';
+import devReliquary from '../images/builders/The_Relicans.png';
+import onlineTeach from '../images/builders/remote-unsplash.jpg';
+import styles from './builders.module.scss';
+
+const DeveloperChampionPage = () => {
+  return (
+    <>
+      <SEO />
+      <PageLayout type={PageLayout.TYPE.SINGLE_COLUMN}>
+        <PageLayout.Header title="New Relic Builders" />
+        <PageLayout.Content>
+          <section className={cx(styles.section, styles.twoColumn)}>
+            <div>
+              <p>
+                {' '}
+                Our goal with the New Relic Community, also known as{' '}
+                <a href="https://www.therelicans.com/">The Relican's</a>, is to
```

Comment:
It seems like we're styling this without a `'`.
```suggestion
                <a href="https://www.therelicans.com/">The Relicans</a>, is to
```